crawler/sql_queries.py

```diff
@@ -83,8 +83,22 @@
 filtered_positive_version = %(filtered_positive_version)s,
 filtered_positive_timestamp = %(filtered_positive_timestamp)s
 WHERE mongodb_id = %(mongodb_id)s
+AND root_sample_id = %(root_sample_id)s
+AND rna_id = %(rna_id)s
+AND result = %(result)s
```

Comment:
Why was it required to add these identifying conditions here? Is `mongodb_id` not indexed and unique?

Reply:
This was incorrectly committed -  we initially added these to see if it would speed up the query. Now removed